tests/test_track_ensemble.py

```diff
@@ -23,7 +24,7 @@ def test_monovariate_besttrack_ensemble():
         output_directory.mkdir(parents=True, exist_ok=True)
 
     perturber = VortexPerturber(
-        storm='al062018', start_date='20180911', end_date=None, file_deck=ATCF_FileDeck.b,
+        storm='al062018', start_date='20180911', end_date=None, file_deck='b',
```

Comment:
this allows the file deck input to be a single letter, or alternatively the name (`BEST`, `OFCL`, etc.), along with the option to directly call the enum value